src/android_utils.py

```diff
@@ -370,3 +370,43 @@ def listdir(path=None):
 
 def apply_android_workarounds():
     _android11_ext_storage_workarounds()
+
+
+class StartupState:
+    FIRST_TIME = 1
+    USB = 2
+    NETWORK = 3
```

Comment:
Is better to use Python enums. Even better with auto enum:
```suggestion
from enum import auto, Enum
class StartupState(Enum):
    FIRST_TIME = auto()
    USB = auto()
    NETWORK = auto()
```